src/inputs/IconCardGroup.tsx

```diff
@@ -0,0 +1,81 @@
+import { ReactNode } from "react";
+import { useCheckboxGroup } from "react-aria";
+import { useCheckboxGroupState } from "react-stately";
+import { Css } from "src/Css";
+import { IconProps } from "src/components/Icon";
+import { Label } from "src/components/Label";
+import { PresentationFieldProps, usePresentationContext } from "src/components/PresentationContext";
+import { useTestIds } from "src/utils";
+import { IconCard } from "src/inputs/IconCard";
+import { HelperText } from "src/components/HelperText";
+import { ErrorMessage } from "./ErrorMessage";
+
+export interface IconCardGroupItemOption {
+  icon: IconProps["icon"];
+  label: string;
+  disabled?: boolean;
+  /** The value of the IconCardGroup item, stored in value array in state. */
+  value: string;
+}
+
+export interface IconCardGroupProps extends Pick<PresentationFieldProps, "labelStyle"> {
+  label: string;
+  /** Called when a card is selected */
+  onChange: (values: string[]) => void;
+  /** Options for the cards contained within the IconCardGroup. */
+  options: IconCardGroupItemOption[];
+  /** The values currently selected. */
+  values: string[];
+  errorMsg?: string;
+  helperText?: string | ReactNode;
+  columns?: number;
+  disabled?: boolean;
+}
+
+export function IconCardGroup(props: IconCardGroupProps) {
+  const { fieldProps } = usePresentationContext();
+  const {
+    options,
+    label,
+    labelStyle = fieldProps?.labelStyle ?? "above",
+    values,
+    errorMsg,
+    helperText,
+    columns = 1,
```

Comment:
Question: what do you think about just using a flex container since the box sizes are fixed you'd still get an even grid, but then the columns would be responsive based on the parent container size 🤔

Reply:
Sure, that was my idea at first and then I just tried to match the `CheckboxGroup` but, yeah I'll leave it as a flex with a flex-wrap